configuring.tex

```diff
@@ -1,424 +1,185 @@
-    or \screentext{DVI ONLY} video output with no audio. \newline
-  \item \screentext{CRT EMULATION}
-    selects whether CRT scanline emulation should be applied to the video output or not.
-\end{itemize}
 
-\subsection{Audio}
+When prompted, reboot the machine.\footnote{If you select a core that does not have {\tt MEGA65.ROM} as one of the embedded files, the utility will prompt you to move the SD card to your PC to copy this file onto it. This only happens when using a MEGA65 core from somewhere other than an official MEGA65 release package. For more information about cores and obtaining {\tt MEGA65.ROM}, see chapter \vref{cha:cores}.}
 
-\begin{center}
-\includegraphics[width=0.7\linewidth]{images/ss-m65config-4.png}
-\end{center}
+\subsection{Obtaining the Bundled Software}
 
-\begin{itemize}
-  \item \screentext{AUDIO OUTPUT} selects whether the SIDs and digital audio channels are combined to provide a monaural signal or whether the left and right tagged audio sources are separated to provide a stereo signal. This setting can be changed in the Audio Mixer of the Freeze Menu, or under the control of MEGA65-enabled software.
-  \item \screentext{SWAP STEREO CHANNELS} allows switching the left and right-hand sides of the stereo audio output. This is useful for software that expects left and right SIDs to be at swapped addresses compared with the MEGA65 defaults.
-  \item \screentext{DAC ALGORITHM} allows selecting between two different digital to analog conversion algorithms. Both options sound good and the selection is a personal preference.
-  \item \screentext{AUDIO AMPLIFIER} allows enabling or disabling the audio amplifier contained in some models of the MEGA65. This option works for audio outputs, e.g., internal speaker or loud speaker.
-\end{itemize}
+The system files copied to the freshly formatted SD card do not include the bundled software that was included with the original SD card in the internal card slot. You can use your PC to copy these files off of the original SD card, then copy them back onto the new SD card.
 
-\subsection{Network}
-
-\begin{center}
-\includegraphics[width=0.7\linewidth]{images/ss-m65config-5.png}
-\end{center}
+The MEGA65 Filehost website\index{Filehost website} hosts all manner of files you can download for your MEGA65. This includes the latest versions of the platform components, alternate cores, and hundreds of games, demos, and applications produced by the MEGA65 community. This also includes the bundled software from the original SD card included with the computer.
 
-\begin{itemize}
-  \item \screentext{MAC ADDRESS} allows you to set the default MAC address of your MEGA65. This can be changed at run-time by MEGA65-enabled software.
-\end{itemize}
+If you no longer have the bundled software files, you can obtain them from the MEGA65 Filehost website. Visit the following URL, then search for ``MEGA65 Release SD Card - Intro Disk Extras.''
 
-% 2021-03-17 edits by SBC
+\url{https://files.mega65.org}
```

Comment:
Maybe! That'd work in this case, but not as well in other cases where the documentation instructs the reader to find files on Filehost. Filehost does not (yet?) allow aggregating software for Windows/Mac/Linux under a single detail page, so later instructions tell the user to search for a title instead of offering three short links.